### PR TITLE
docs(installing_deis): add Engine Yard

### PR DESCRIPTION
**Pull request by carmstrong**

### CODE REVIEW

File: docs/installing_deis/index.rst [hunk 1]
```diff
@@ -22,13 +22,14 @@ information on how to get set up with CoreOS.
     quick-start
     system-requirements
     aws
-    azure
+    baremetal
     digitalocean
+    engineyard
     gce
+    azure
```
Comment (mboersma) [inline docs/installing_deis/index.rst]: I had to build this PR to see how this alphabetizes things correctly: "Microsoft Azure"

**Comment by mboersma** [comment]
LGTM. I built and browsed the docs locally.

File: docs/installing_deis/index.rst (hunk 1 above)
Comment (carmstrong) [inline docs/installing_deis/index.rst]: Yep :)

**Comment by bacongobbler** [comment]
LGTM

**Comment by bacongobbler** [comment]
failure is unrelated and the doc tests passed so I think this is good to :ship:

**Comment by carmstrong** [comment]
> so I think this is good to

Yep, I'm waiting to merge this until Monday.

**Comment by mboersma** [comment]
Needs rebase.

**Pull request merged**